lib/fpm/package/python.rb

```diff
@@ -240,7 +240,7 @@ def load_package_info(setup_py)
         next if attributes[:python_disable_dependency].include?(name)
 
         # convert == to =
-        if cmp == "=="
+        if cmp == "==" or cmp == "~="
```

Comment:
https://www.python.org/dev/peps/pep-0508/#grammar

Reply:
it still not clear to me what wrong with this code?

Reply:
ok, if you want me to change something just please give me to know,
this change been done just because of python azure 3.0.0 module